rpc/server.go

```diff
@@ -861,6 +881,34 @@ func (ss *simpleServer) Stop() error {
 	return err
 }
 
+type SimpleServerStats struct {
```

Comment:
Same comment about lint and comments here.

Do types returned by `Stats` methods need to be exported for FTDC's reflection to work? If not, should we just keep all of this API private since it is all within the `rpc` package?